Core/KSPManager.cs

```diff
@@ -151,6 +152,76 @@ public KSP AddInstance(KSP ksp_instance)
             return ksp_instance;
         }
 
+        /// <summary>
+        /// Clones an existing KSP installation.
+        /// </summary>
+        /// <param name="existing_instance">The KSP instance to clone.</param>
+        /// <param name="new_name">The name for the new instance.</param>
+        /// <param name="new_path">The path where the new instance should be located.</param>
+        public void CloneInstance (KSP existing_instance, string new_name, string new_path)
+        {
+            if (existing_instance.Valid)
+            {
+                KspVersion version = existing_instance.Version();
+
+                CKAN.DLC.MakingHistoryDlcDetector dlcDetector = new DLC.MakingHistoryDlcDetector();
+                bool DLC = dlcDetector.IsInstalled(existing_instance, out string identifier, out UnmanagedModuleVersion moduleVersion);
+                string dlcVersion = moduleVersion.ToString();
+
+                FakeInstance(new_name, new_path, version, DLC, dlcVersion);
```

Comment:
Do we actually want "cloning" to create a fake instance? I would have thought the point of it would be to copy a real instance, e.g. clone my Steam install so I could put different mods in the copy.

Reply:
Oops, might have misunderstood that one.
So if we really copy an existing one, what should we copy? The whole directory with all of KSPs files?

Reply:
I would think we would copy just enough to create a pristine working copy, but let's let @politas weigh in.

Reply:
I meant copy the whole directory with all files, so that users can then trial some new mods without messing up their main install. Or create a base install with a minimal set of core mods, and clone that to try different planet packs or strategy replacements, or tech trees or whatever.

Reply:
Okay, gotcha.